### PR TITLE
Fixed the wrong matching title in the sidebar and section

### PR DESCRIPTION
**Pull request by bopardikarsoham**
The wrong matching title in the sidebar and section has been corrected as per #715.

### CODE REVIEW

**Comment by bopardikarsoham** [comment]
I believe the branch has now been updated. I can only see the adjustment I made for this problem. @Tansito, the adjustments I made to the earlier PR have now been merged.

**Review by Tansito — APPROVED** [review]

File: notebooks/ch-algorithms/superdense-coding.ipynb [hunk 1]
```diff
@@ -2484,7 +2484,7 @@
    "provenance": []
   },
   "kernelspec": {
-   "display_name": "Python 3 (ipykernel)",
+   "display_name": "Python 3",
```
Comment (eddybrando) [inline notebooks/ch-algorithms/superdense-coding.ipynb:2487]: I believe this change isn't related to https://github.com/Qiskit/platypus/issues/715.

File: notebooks/ch-algorithms/superdense-coding.ipynb [hunk 2]
```diff
@@ -2498,7 +2498,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.7"
+   "version": "3.8.5"
```
Comment (eddybrando) [inline notebooks/ch-algorithms/superdense-coding.ipynb:2501]: I believe this change isn't related to https://github.com/Qiskit/platypus/issues/715.

**Review by eddybrando — COMMENTED** [review]
We should only introduce changes related to the issue being solved.

File: notebooks/ch-algorithms/superdense-coding.ipynb (hunk 2 above)
Comment (Tansito) [reply notebooks/ch-algorithms/superdense-coding.ipynb:2501]: Totally agree with you. I talked with @frankharkins about these changes before merge it and they seemed to be metadata related with the environment where the notebook was edited so I saw it fine but good point for next time.

Comment (bopardikarsoham) [reply notebooks/ch-algorithms/superdense-coding.ipynb:2501]: Oh yeah, the Python version has been downgraded for this notebook @eddybrando

**Pull request merged**